superset-frontend/src/cccs-viz/plugins/plugin-chart-application-links/src/ApplicationLinks.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { ApplicationsProps } from './types';
+
+export default function ApplicationLinks(props: ApplicationsProps) {
+  const { application, appVal, appType } = props;
+  const [alfredCount, setAlfredCount] = useState(-1);
+
+  let url = '';
+  let infoType = '';
+  let callback_url = '';
+
+  if (application === 'ALFRED') {
+    if (appType === 'USER_ID') {
+      url = `https://alfred-tst.u.chimera.azure.cyber.gc.ca/?expression=MATCH%20(email:EMAIL_ADDRESS)%20WHERE%20email.value%20in%20[%22${appVal}%22]%20return%20email.value,%20email.maliciousness,%20email.uri`;
+      infoType = 'user id';
+      callback_url = 'user_id';
+    } else {
+      url = `https://alfred-tst.u.chimera.azure.cyber.gc.ca/?expression=MATCH%20(ip%3AIP_ADDRESS)%20WHERE%20ip.value%20IN%20%5B%22${appVal}%22%5D%20RETURN%20ip.value%2C%20ip.maliciousness%2C%20ip.creation_date%2C%20ip.created_by%2C%20ip.uri%2C%20ip.report_uri`;
+      infoType = 'IP';
+      callback_url = 'ip_string';
+    }
+  }
+
+  useEffect(() => {
+    fetch(
+      // eslint-disable-next-line no-restricted-globals
+      `http://${location.host}/api/v1/proxy/alfred/${callback_url}/${appVal}`,
+    )
+      .then(res => res.json())
+      .then(response => {
+        if (response !== null && response.payload !== undefined) {
+          setAlfredCount(response.payload.data?.length);
+        } else {
+          setAlfredCount(0);
+        }
+      })
+      .catch(e => {
+        // eslint-disable-next-line no-console
+        console.log(e);
```

Comment:
This is just logging any error we get back from alfred when we request data.  The message the user sees is that Alfred has no results.  If they know that there are results then we can quickly get the message from the UI.  I'm not sure we have anywhere else to log this type of condition in a way that is easily accessible.

Reply:
The other option would be to revisit this when we do the rest of the applications and have a friendly way to show the user there is an error along with a way for them to retrieve that error.  This can be done in the future though and not as part of this ticket.